lib/css/components/select.less

```diff
@@ -0,0 +1,194 @@
+.s-select {
+    --_se-arrow-bc: currentColor transparent;
+    --_se-arrow-size: var(--su-static4); // Constant // TODO create name/styling convension for constants
+    --_se-select-bc: var(--bc-darker);
+    --_se-select-bc-focus: var(--theme-secondary-300);
+    --_se-select-bs-focus: 0 0 0 var(--su-static4) var(--focus-ring);
+    --_se-select-bg: var(--white);
+    --_se-select-br: var(--br-sm);
+    --_se-select-fc: var(--black);
+    --_se-select-px: 0.7em;
+    --_se-select-py: 0.6em;
+    --_se-select-fs: var(--fs-body1);
+
+    // CONTEXTUAL STYLES
+    @supports (-webkit-overflow-scrolling: touch) {
+        --_se-select-fs: 16px; // Increase font size for mobile safari. This keeps our inputs from zooming the page while focused
+        --_se-select-px: 0.55em; // Compensate for the larger font size so we generally keep the input the same size
+        --_se-select-py: 0.4em; // Compensate for the larger font size so we generally keep the input the same size
+    }
+
+    // MODIFIERS
+    .is-disabled &,
+    .is-readonly &,
+    .has-success &,
+    .has-error &,
+    .has-warning & {
+        position: relative;
+    }
+
+    .is-disabled & {
+        --_se-arrow-bc: var(--bc-darker) transparent;
+    }
+
+    &&__sm {
```

Comment:
Sizing could also likely benefit greatly from a mixin that is shared between all inputs

Reply:
I added a mixin in this commit https://github.com/StackExchange/Stacks/pull/1184/commits/718c52ee7e194d55b04f313349f61ad91613d047. I went pretty programmatic since we're using pseudo-private custom properties. It came out looking a little wacky, though I think the mixin is pretty effective. Let me know if it's too wacky (or not wacky enough).